app/assets/javascripts/discourse/app/templates/discovery/topics.hbs

```diff
@@ -62,6 +62,22 @@
       </div>
     {{/if}}
   {{/if}}
+
+  {{#if
```

Comment:
I would argue that in this kind of very HOT templates, it would be better to be as logic less as possible, so maybe:

```
{{#if this.shouldRenderNewListHeader}}
  ...
{{/if}}
```